distributor-widget/integrations.md

```diff
@@ -119,6 +123,15 @@ If a hotel is selected, information about it is also added to the event. \(Note:
 
 Some events expose additional data layer variables. They are described separately for each event.
 
+##### TrackingConsents <a id="trackingconsents"></a>
```

Comment:
I'm not sure what's the decision about these links? It makes sense to me to define the, so in the future when we change the heading, we don't break the links in case some other website is already using them. @dmarchuk @dobras

Reply:
I am not using them by default. There is actually now way of not breaking them if you want to keep them up to date with names of entities. Once you rename entity, you should rename the link ergo breaking it to the others
Also i have this issue: https://mews.myjetbrains.com/youtrack/issue/TDD-1726
and i will be removing them.

Reply:
will remove them in the new sections

Reply:
I will do it as a part of that issue. In new ones just dont use them :D.